learninGitHub101.txt

```diff
@@ -1,3 +1,5 @@
 This is Roy, I created this text file on July 17 2017 at 1014 AM pacific standard time.
 
-Please checkout this branch & file, edit it on a new line, save the file up, git commit, and then git push to commit it to the repository.
+Please clone this branch, edit it on a new line, save the file up, git commit, and then git push to commit it to the repository.
```

Comment:
No worries, when you're ready the "master" will appear. I'll walk you through using one of those screen share thingies and do it hands on & live. 👍